mdk/README.txt

```diff
@@ -33,14 +36,14 @@ If you preffer to use IntelliJ, steps are a little different.
 
 Step 4: The final step is to open Eclipse and switch your workspace to /eclipse/ (if you use IDEA, it should automatically start on your project)
 
-If at any point you are missing libraries in your IDE, or you've run into problems you can run "gradlew --refresh-dependencies" to refresh the local cache. "gradlew clean" to reset everything {this does not effect your code} and then start the processs again.
+If at any point you are missing libraries in your IDE, or you've run into problems you can run "gradlew --refresh-dependencies" to refresh the local cache. "gradlew clean" to reset everything {this does not affect your code} and then start the processs again.
 
 Should it still not work, 
 Refer to #ForgeGradle on EsperNet for more information about the gradle environment.
 
 Tip:
 If you do not care about seeing Minecraft's source code you can replace "setupDecompWorkspace" with one of the following:
-"setupDevWorkspace": Will patch, deobfusicated, and gather required assets to run minecraft, but will not generated human readable source code.
+"setupDevWorkspace": Will patch, deobfuscated, and gather required assets to run minecraft, but will not generated human readable source code.
```

Comment:
Shouldn't it be deobuscat and generat rather than deobfuscated an generated?

Reply:
`deobfuscate`. Use a verb.